Source/Advance.cpp

```diff
@@ -131,7 +131,7 @@ PeleC::do_mol_advance(
   for (int n = 0; n < src_list.size(); ++n) {
     if (
       src_list[n] != diff_src
-#ifdef AMREX_PARTICLES
+#ifdef PELEC_SPRAY
```

Comment:
Instead of having these ifdefs here, could we just always have `spray_src` in the enum and just keep the if condition? And then kinda like `les_src`, `spray_src` only gets `define`d if asked for? This would remove a whole bunch of ifdefs that are interleaved into if conditions.